Makefile

```diff
@@ -1,11 +1,14 @@
+heroku:
+	go build cmd/neighbors.go
+
 compile-win: clean-win
-	go-bindata -pkg assets -o assets.go assets/... && MOVE assets.go pkg/assets && go build cmd/neighbors.go
+	go-bindata -pkg assets -o assets.go assets/... && MKDIR pkg\assets && MOVE assets.go pkg\assets\assets.go && go build cmd/neighbors.go

```

Comment:
Does something not work if you normalize the slashes?

Reply:
yeah go bin data uses linux file separators